akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -236,7 +294,7 @@ private[akka] object StreamLayout {
       if (Debug) validate(this)
 
       CompositeModule(
-        subModules = Set(this),
+        Set(this),
```

Comment:
because it didn’t make sense for me to special-case this single parameter-passing: other parameters are also not named, and in fact we only routinely name boolean parameters